content/applications/inventory_and_mrp/manufacturing/workflows/subcontracting_basic.rst

```diff
@@ -0,0 +1,200 @@
+factored in when calculating the product's expected arrival date.
+
+.. note::
+   When using basic subcontracting, there is no need to list components in the :guilabel:`Components`
+   tab of the |BoM|, since the components required for manufacturing, and the means by which they
+   are acquired, are handled by the subcontractor.
+
+Basic subcontracting workflow
+=============================
+
+The basic subcontracting workflow consists of up to four steps:
+
+#. Create a sales order (SO) for the subcontracted product; doing so creates a |PO| to purchase the
+   product from the subcontractor.
+#. Confirm the |PO| created in the previous step, or create a new |PO|; doing so creates a receipt
+   order or a dropship order.
+#. Process the receipt once the subcontractor has finished manufacturing the subcontracted product,
+   and shipped it back to the contracting company, **OR** process the dropship order to ship the
+   product directly to the customer.
+#. If the workflow was started by creating an |SO|, and the finished product is not dropshipped to
+   the end customer, process the delivery order once the product is shipped to the customer.
+
+The specific number of steps depends on the reason that the subcontracted product is being purchased
+from the subcontractor.
+
+If the reason is to fulfill a specific customer order, the process starts with creating an |SO|, and
+ends with delivering the product to the customer, or having the subcontractor dropship it to them.
+
+If the reason is to increase the quantity of stock on-hand, the process starts with creating a |PO|,
+and ends with receiving the product into inventory.
+
+Create SO
+---------
+
+It is only necessary to complete this step if the product is being purchased from the subcontractor
+to fulfill a customer need. If the product is being purchased to increase the quantity of stock
+on-hand, move on to the next step.
+
+To create a new |SO|, navigate to :menuselection:`Sales app --> Orders --> Orders`, and click
+:guilabel:`New`.
+
+Select the customer in the :guilabel:`Customer` drop-down menu. Then, click :guilabel:`Add a
+product` on the :guilabel:`Order Lines` tab, select a subcontracted product in the
+:guilabel:`Product` drop-down menu, and enter a quantity in the :guilabel:`Quantity` field.
+
+Click :guilabel:`Confirm` to confirm the |SO|, at which point a :guilabel:`Purchase` smart button
+appears at the top of the page. This opens the |PO| created to purchase the subcontracted product
+from the subcontractor.
+
+.. note::
+   An |SO| for the product only creates a |PO| if the *Replenish on Order (MTO)* route is enabled on
+   the product's page, **and** there is not enough stock of the product on-hand to fulfill the |SO|.
+
+   If there is enough stock on-hand, confirming an |SO| for the product instead creates a delivery
+   order, because Odoo assumes that the |SO| is fulfilled using the stock in the warehouse.
```

Comment:
```suggestion
   If there is enough stock on-hand, confirming an |SO| for the product creates a delivery
   order instead, because Odoo assumes that the |SO| is fulfilled using the stock in the warehouse.
```